### DISCUSSION TITLE
Wondrous prompt

### DISCUSSION THREAD

**Discussion by stefanfrede** [starship/starship#3501] [Q&A] [answered]
Hi!

From time to time I get the following prompt and I can't figure out what it is trying to tell me:

[![wondrous prompt.png](https://i.postimg.cc/y6QBb0HT/wondrous-prompt.png)](https://postimg.cc/7b2pCGbC)

Can someone please enlighten me. Thank you!

**Comment by chipbuster** [comment, marked as answer]
This is the jobs module--it tells you you have background jobs running.

If you didn't launch them yourself, what's likely happening is that there's some other shell framework (e.g. fzf, z.sh, autojump) that's spawning temporary jobs that starship is picking up on.
